### ISSUE TITLE
[Linux] [OpenGL] Maximizing the Cemu window makes the whole screen dark

### ISSUE THREAD

**Issue by uranuspucksaxophone** [cemu-project/Cemu#222]
In full-screen mode this does not happen, the game is displayed correctly.

**Comment by LeTeTeuX** [comment]
Can confirm that this also happens on Windows, on both 2.0 and 1.27 versions, but not in 1.26.2. This also happens if you try to resize the window, then it goes completely dark no matter what, unless you go on fullscreen mode.

**Comment by Crementif** [comment]
I checked the SVN repository and if I had to guess (not 100% certain, but it was a pretty major change made on march 12th which seems to match the dates). So if anyone wants to look at what might be wrong:

Exzap basically switched from using Cemu's own GLContext wrapper to wxWidgets cross-platform one (which was kinda broken when the OpenGL stuff was written a long time ago). Probably an initialization bug or something where the OpenGL swapchains aren't (re)sized correctly causing it to black screen if I had to guess.
